vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html

```diff
@@ -779,19 +779,26 @@
           this.contentDocument.defaultView.addEventListener('resize', function(e) {
             _this._grid.redraw();
           });
-          _this._grid.redraw();
+
+          _this.then(function() {
+            // ...because one then just isn't enough.
+            _this.then(function() {
+              _this._grid.redraw();
+            });
+          });
         });
         this.$.measureobject.data = "about:blank";
       },
 
       attached: function() {
-        this._grid.attached(this, Polymer.dom(this).querySelector("table"), Polymer.dom(this.root));
+        var _this = this;
+        _this._grid.attached(this, Polymer.dom(this).querySelector("table"), Polymer.dom(this.root));
```

Comment:
Here is where I meant with _this and this, I like to see _this in nested functions, not in the same level where _this is defined.